catalogue/webapp/pages/work.js

```diff
@@ -186,13 +247,147 @@ export const WorkPage = ({ work }: Props) => {
+                          workId={work.id}
+                          downloadOptions={downloadOptions}
+                        />
+
+                        {!(downloadOptions.length > 0) &&
+                          sierraIdFromManifestUrl &&
+                          childManifestsCount === 0 && (
+                            <NextLink
+                              {...downloadUrl({
+                                workId: work.id,
+                                sierraId: sierraIdFromManifestUrl,
+                              })}
+                            >
+                              <a>Download options</a>
+                            </NextLink>
+                          )}
+                      </Space>
+                      {license && (
+                        <WorkDetailsText
+                          title="License"
+                          text={[license.label]}
+                        />
+                      )}
+                    </div>
+                    {license && (
+                      <Space
+                        v={{
+                          size: 'l',
+                          properties: ['margin-top'],
+                        }}
+                      >
+                        <ExplanatoryText
+                          id="licenseDetail"
+                          controlText="Can I use this?"
+                        >
+                          <>
+                            {license.humanReadableText.length > 0 && (
+                              <WorkDetailsText
+                                text={license.humanReadableText}
+                              />
+                            )}
+
+                            <WorkDetailsText
+                              text={[
+                                `Credit: ${work.title.replace(
+                                  /\.$/g,
+                                  ''
+                                )}.${' '}
+                ${
+                  credit
+                    ? `Credit: <a href="https://wellcomecollection.org/works/${work.id}">${credit}</a>. `
+                    : ` `
+                }
+              ${
+                license.url
+                  ? `<a href="${license.url}">${license.label}</a>`
+                  : license.label
+              }`,
+                              ]}
+                            />
+                          </>
+                        </ExplanatoryText>
+                      </Space>
+                    )}
+                  </Space>
+                </Layout12>
+              </>
+            )}
+            <WorkDetails
+              work={work}
+              itemUrl={itemUrlObject}
+              iiifPresentationManifest={iiifPresentationManifest}
+              childManifestsCount={childManifestsCount}
+              imageCount={imageTotal}
+              showAvailableOnline={availableOnline}
+            />
+          </>
+        )}
```

Comment:
Are all these components part of a section? And if so could the be wrapped up in a component?